assets/js/stripe-payment-request.js

```diff
@@ -490,6 +424,148 @@ jQuery( function( $ ) {
 			} );
 		},
 
+		/**
+		 * Creates stripe paymentRequest element or connects to custom button
+		 *
+		 * @param {object} elements       Stripe elements instance.
+		 * @param {object} paymentRequest Stripe paymentRequest object.
+		 *
+		 * @return {object} Stripe paymentRequest element or custom button jQuery element.
+		 */
+		createPaymentRequestButton: function( elements, paymentRequest ) {
+			var button;
+			if ( wc_stripe_payment_request_params.button.is_custom ) {
+				button = $( wc_stripe_payment_request_params.button.css_selector );
+				if ( button.length ) {
+					// We fallback to default paymentRequest button if no custom button is found in the UI.
+					// Add flag to be sure that created button is custom button rather than fallback element.
+					button.isCustom = true;
```

Comment:
While using a property of the `button` object makes a lot of sense, it goes against jQuery common practices, which is to manipulate the DOM element instead of the jQuery object. I think it would be better to use `button.data( 'isCustom', true )` as a setter and `button.data( 'isCustom' )` as a getter.

Reply:
Good point, thank you! Fixed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1197/commits/9702bf469a1160463d0bbf0b5ad23ed34c17d0a1